velox/core/QueryConfig.h

```diff
@@ -411,10 +416,15 @@ class QueryConfig {
   }
 
   uint64_t maxSpillRunRows() const {
-    static constexpr uint64_t kDefault = 12 * 1024 * 1024;
+    static constexpr uint64_t kDefault = 12L << 20;
     return get<uint64_t>(kMaxSpillRunRows, kDefault);
   }
 
+  uint64_t maxSpillBytes() const {
+    static constexpr uint64_t kDefault = 100L << 40;
```

Comment:
I think Java is 100GB by default and let's keep it the same for now. If it is zero, then there is no limit? Thanks!